modules/ROOT/pages/import-api-specification-design-center.adoc

```diff
@@ -0,0 +1,44 @@
+= Importing an API Specification from Design Center
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API Definition from Design Center into either into a new Mule Project, or an existing one.
+
+This task only works for RAML types of API specifications.
+
+== Importing an API from Design Center Into a New Project
```

Comment:
This subtitle and the next are not exactly similar: do we mean to make a distinction between importing an API and importing an API specification? 
If these mean the same thing, consider searching these topics to standardize on one, best wording for this.

Reply:
Is there anything to say here about new project vs. existing project: pros and cons?